packages/system/hetzner-robotlb/Makefile

```
@@ -0,0 +1,9 @@
+export NAME=hetzner-robotlb
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)